app/konnect/getting-started/index.md

```diff
@@ -2,60 +2,50 @@
 title: Get started with Konnect
 ---
 
-## Quickstart guide
+{% mermaid %}
+flowchart TD
+    A{Start my {{site.konnect_short_name}} journey} -->B(Set up an API for me)
+    A{Start my {{site.konnect_short_name}} journey} -->C(I want to do it myself)
+    B --> D(Use the onboarding wizard for the first time)
```

Comment:
❗ For reviewers: For this GA, you can only do the wizard once, you can't click the link and do it again. I'm trying to find a good way to word this and this is all I came up with. Any suggestions would be great!

Reply:
Hmm. Is this a failing of the UI that we're covering for, though? I think we should just do "Use the onboarding wizard" and not have the rest.

 If you want, we could leave a footnote after the workflow diagram that explains the limitation, but i don't think it should be in the diagram.